### PR TITLE
Fixes swagger-editor/#1502.

### PR DESCRIPTION
**Pull request by owenconti**
Fixes https://github.com/swagger-api/swagger-editor/issues/1502.
Fixes https://github.com/swagger-api/swagger-ui/issues/3385.
Fixes https://github.com/swagger-api/swagger-ui/issues/2700.

Change logic for markdown rendering to:

1. Convert source markdown to HTML
2. Sanitize HTML
3. Send sanitized HTML to markdown renderer

Works with the following test specs:

https://gist.github.com/tbarsballe/bb135cd6f65b1341bd4184f003ca8f4d
https://gist.github.com/shockey/e65144c103f4cf3cd501d2c2d1c63b18
https://gist.github.com/shockey/7346f22b0d31b39ae65b2b5fe39a8f51

### CODE REVIEW

**Comment by owenconti** [comment]
I'm going to add various XSS cases to the existing sanitization tests.

File: package.json [hunk 1]
```diff
@@ -72,6 +73,7 @@
     "redux": "^3.x.x",
     "redux-immutable": "3.0.8",
     "redux-logger": "*",
+    "remarkable": "^1.7.1",
```
Comment (shockey) [inline package.json]: Since we're using Remarkable directly, do we need `react-remarkable` anymore?

Comment (owenconti) [reply package.json]: Nope, I think we can remove it. I also want to try removing `ReactMarkdown` from the OAS3 component and replace it with `dangerouslySetInnerHTML` like in the 2.0 component.

**Comment by shockey** [comment]
Git merge = 💥

**Comment by shockey** [comment]
Okay, got it working again 😄

**Comment by shockey** [comment]
This fixes https://github.com/swagger-api/swagger-ui/issues/3385 as well.

**Comment by shockey** [comment]
Merged so we can release this fix.

I've opened a new issue for the tests that you mentioned, @owenconti.

**Comment by ettaramesh** [comment]
Its been mentioned that #3385 is being fixed as part of this issue. Could you please let me know which version of swagger-ui has this fix available. I have tested with 3.33.0 and the issue still exists.

**Pull request merged**